src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1516,4 +1518,23 @@ public void validateDeleteRequest(Dataset dataset, List<DataDeletionTableModel>
             }
         }
     }
+
+    private static final String rowCountTemplate = "SELECT COUNT(<rowId>) FROM `<project>.<dataset>.<table>`";
```

Comment:
I think `COUNT(1)` would also work, without scanning data.

Reply:
I think the `1` refers to a column, but I could be wrong. In any case, it won't stop a scan in this case, because we want the count of the snapshot view.

Reply:
Ah right, forgot this is targeting a view that already mentions the column. Oh well.

In case it's helpful for anything in the future, I think `COUNT(1)` does give some "magic" savings in terms of bytes scanned when targeting a real table:
![image](https://user-images.githubusercontent.com/2755881/82951730-bc751080-9f75-11ea-8817-c643ed1949fb.png)
Probably wouldn't make a meaningful difference either way unless we were counting a V2F-scale table multiple times per day.